packages/build-utils/src/convert-runtime-to-plugin.ts

```diff
@@ -44,11 +44,11 @@ export function convertRuntimeToPlugin(
         },
       });
 
-      functionsManifest[entrypoint] = {
+      pages[entrypoint] = {
         handler: output.handler,
         runtime: output.runtime,
-        memory: config.memory || output.memory,
-        maxDuration: config.maxDuration || output.maxDuration,
+        memory: output.memory,
+        maxDuration: output.maxDuration,
```

Comment:
Some of this logic was moved into `updateFunctionsManifest()` so it can be shared with `vercel-plugin-node`